src/NServiceBus.RavenDB/Outbox/OutboxPersister.cs

```diff
@@ -25,7 +26,7 @@ public bool TryGet(string messageId, out OutboxMessage message)
             {
                 session.Advanced.AllowNonAuthoritativeInformation = false;
                 // We use Load operation and not queries to avoid stale results
-                result = session.Load<OutboxRecord>(GetOutboxRecordId(messageId));
+                result = session.Load<OutboxRecord>(new[] { GetOutboxRecordId(messageId), GetOutboxRecordIdWithoutEndpointName(messageId) }).FirstOrDefault(o => o != null);
```

Comment:
In the scenario of some endpoints not yet upgraded to the hotfix, both versions of the document could exist, so I made sure the first one checked is the new format.